lib/stdlib/src/erl_eval.erl

```diff
@@ -88,8 +90,8 @@
       NewBindings :: binding_struct()).
 exprs(Exprs, Bs) ->
     case check_command(Exprs, Bs) of
-        ok -> 
-            exprs(Exprs, Bs, none, none, none);
+        ok ->
+            exprs(Exprs, Bs, none, none, none, fuv());
```

Comment:
I went with `empty_fun_used_vars()` because I have been bitten in the past when I had to change the default value of something and that led to me chasing which of all empty maps in a file were the empty maps I was looking for. :D But if you prefer `#{}`, then I will gladly change it.

Reply:
Fine with me :)